### DISCUSSION TITLE
Quotas

### DISCUSSION THREAD

**Discussion by SuleGen** [ThioJoe/YT-Spammer-Purge#252] [Q&A] [answered]
How can I switch projects? I made a new project on the google cloud console and followed all of the steps like last time but it's still using the other project.

**Comment by ThioJoe** [comment, marked as answer]
Each project has it's own client_secrets.json. So to use a different project, replace the old client_secrets with the new one from the new project. You can also keep a copy of one for each project and rename it to like client_secrets2.json and rename it when you want to use it.

**Reply by SuleGen** [reply]
How can I replace the old project?
(On Mac by the way)

**Reply by kiatchun** [reply]
just delete client_secrets.json and token.pickle and replace with the new one.

**Reply by SuleGen** [reply]
Alright i got it working thanks for the help.
